source/tutorials/monitor-minio-tenant.rst

```diff
@@ -0,0 +1,122 @@
+======================
+Monitor a MinIO Tenant
+======================
+
+.. default-domain:: minio
+
+.. contents:: Table of Contents
+   :local:
+   :depth: 1
+
+
+This procedure documents monitoring a MinIO object storage tenant deployed on
+|vcf| 4.2.
+
+Monitor MinIO using Prometheus
+------------------------------
+
+MinIO exports `Prometheus <https://prometheus.io/>`__ compatible data as an
+authorized endpoint. Users can monitor MinIO tenants using any
+Prometheus-compatible application by scraping data from this endpoint.
+
+1) Generate a Bearer Token
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The MinIO scrape endpoint requires authentication. Prometheus supports a 
+bearer token approach to authenticate scrape requests. Use the 
+`mc admin prometheus generate` command to generate the required bearer token.
```

Comment:
If the customer uses the latest version of `mc` it will print URL for `v2` where as the default version of MinIO with vDDp will only support legacy.

Reply:
Oof. So we need to warn that users should not use a newer version of `mc` than their `minio` tenant is using?

Reply:
I will make a change to mc to print the right information based on MinIO version.. this way the customer is not left guessing. This doc should be fine.